org.meshtastic.meshtasticd.yaml

```diff
@@ -0,0 +1,67 @@
+id: org.meshtastic.meshtasticd
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: meshtasticd-wrapper.sh
+modules:
+  - name: meshtasticd
+    buildsystem: simple
+    build-options:
+      build-args:
+        # PlatformIO requires internet access at build-time to download pkgs
+        - --share=network
+    build-commands:
+      - platformio run -e native
+      - install -Dm755 .pio/build/native/program /app/bin/meshtasticd
+      # Install the default config files and directories
+      - install -d /app/share/meshtasticd
+      - install -d /app/share/meshtasticd/config.d
+      - install -d /app/share/meshtasticd/available.d
+      - cp -R bin/config.d/* /app/share/meshtasticd/available.d
+      - install -m644 bin/config-dist.yaml /app/share/meshtasticd/config-dist.yaml
+    sources:
+      - type: git
+        url: https://github.com/meshtastic/firmware.git
+        tag: v2.5.18.89ebafc
+        commit: 89ebafc8b8d85ee12002b9c574f8472d99e9ec50
+      - type: patch
+        paths:
+          - patches/stdcppfs.patch
+    modules:
+      - python3-modules.json
+      - modules/yaml-cpp.yaml
+      - modules/libgpiod.yaml
+      - modules/i2c-tools.yaml
+      - modules/microhttpd.yaml
+      - modules/jansson.yaml
+      - modules/orcania.yaml
+      - modules/yder.yaml
+      - modules/ulfius.yaml
```

Comment:
everything but the automatically generated manifest and the shared-modules should be inline and not in separate files. It's unmaintainable otherwise.